Lekce_02_Ukol04/Lucistnik.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Lekce_02_Ukol04
+{
+    internal class Lucistnik
+    {
+        private int _pocetSipu;
+
+        public Lucistnik(int pocetSipu)
+        {
+            if (pocetSipu >= 0)
+            {
+                _pocetSipu = pocetSipu;
+            }
+            else
+            {
+                Console.WriteLine("Počet šípů nemůže být negativní.");
+            }
+        }
+
+        public void Vystrel()
+        {
+            _pocetSipu--;
+        }
```

Comment:
Myslel jsem to nejak takhle :) Muzeme vracet treba bool, jako je to u TryParse. A pokud uz ma lucistnik 0 sipu, tak uz nemuze nic vystrelit a metoda vrati false, jakoze se mu nepovedlo nic vystrelit. Pokud ma ale pozitivni pocet sipu, tak vystreli a vrati se true :) 
```suggestion
        public bool ZkusVystrelit()
        {
           if (_pocetSipu <= 0)
           {
               return false;
           }
           
           _pocetSipu--;
           return true;
        }
```

Reply:
díky moc:)